### PR TITLE
Update top banner for Apex 2022

### PR DESCRIPTION
**Pull request by JakeatRipple**

### CODE REVIEW

File: styles/_top-banner.scss [hunk 1]
```diff
@@ -1,8 +1,7 @@
 .top-banner {
     height: 46px;
-    background: url(../img/backgrounds/bg-apex-banner.svg);
+    background: url(../img/backgrounds/bg-apex-banner-2.gif);
```
Comment (mDuo13) [inline styles/_top-banner.scss]: Why are the new banner backgrounds in GIF format? Wouldn't SVG be preferable for this type of image? We _must_ have access to the source file for this graphic, right? Even if it's only available in raster form, PNG is more compressible with color options that are as good or better than GIF for still images (and these aren't animated).

**Review by mDuo13 — APPROVED** [review]
Looks good, though if we can switch the GIFs out for higher-quality assets first I would prefer that.

File: styles/_top-banner.scss (hunk 1 above)
Comment (JakeatRipple) [reply styles/_top-banner.scss]: Agreed I normally export as svgs.  these were changed after I had the figma file and just sent as gif.  
I can check if they have svgs or the source file for these.

Comment (JakeatRipple) [reply styles/_top-banner.scss]: Found the files in figma. Updated to pngs for now as I was having issues saving as svg.  The bottom left checkered pattern would not show the purple bg.  Would always show white and black.

**Pull request merged**